ui/admin/tests/unit/controllers/scopes/scope/host-catalogs/index-test.js

```diff
@@ -32,5 +75,55 @@ module(
       assert.strictEqual(controller.page, 1);
       assert.strictEqual(controller.search, searchValue);
     });
+
+    test('cancel action rolls-back changes on the specified model', async function (assert) {
```

Comment:
I would like to constructively discuss a bit this approach.

I think this approach is valid, but at the same time I think we should perform assertions within the same store record.

My thinking is next (I can be wrong):
- We retrieve the record.
- We perform changes within the record.
- We assert those changes have been performed.
- We Cancel changes (rollback)
- We assert the previous changes have been rollback within the same record.

In code I am thinking something like this:

```
const hostCatalog = await store.findRecord(
        'host-catalog',
        instances.hostCatalog.id,
);

hostCatalog.name = 'test';
assert.strictEqual(hostCatalog.name, 'test');
await controller.cancel(hostCatalog);
assert.notEqual(hostCatalog.name, 'test');
```

Reply:
Thanks for the feedback, as mentioned above, ill also refactor these tests, and existing tests to use the same instance for both asserting changes.